server/src/main/java/org/elasticsearch/search/sort/SortValue.java

```diff
@@ -47,6 +49,13 @@ public static SortValue from(BytesRef bytes) {
         return new BytesSortValue(bytes);
     }
 
+    /**
+     * Get a {@linkplain SortValue} for data which cannot be sorted.
+     */
+    public static SortValue empty() {
+        return EMPTY_SORT_VALUE;
```

Comment:
We should probably add a few things to `SortValueTests` too - just to quickly catch stuff like serialization errors.